modules/gis/src/lib/flat-geojson-to-binary.ts

```diff
@@ -497,11 +502,11 @@ function makeAccessorObjects(
       positions: {value: polygons.positions, size: coordLength},
       polygonIndices: {value: polygons.polygonIndices, size: 1},
       primitivePolygonIndices: {value: polygons.primitivePolygonIndices, size: 1},
-      triangles: {value: new Uint32Array(polygons.triangles), size: 1},
+      ...(polygons.triangles && {triangles: {value: new Uint32Array(polygons.triangles), size: 1}}),
```

Comment:
Ditto on preferring a separate if statement rather than the clever expression form.